src/auth/appConfig.ts

```diff
@@ -10,28 +10,34 @@ import { DEFAULT_CORE_NODE, DEFAULT_SCOPE, DEFAULT_BLOCKSTACK_HOST } from './aut
  * `window.location.origin` as the app domain.
  * On non-browser platforms, you need to
  * specify an app domain as the first argument.
+ * 
+ * 
  * @type {AppConfig}
+ * 
+ *  * ## How to authenticate a session
+ * 
+ *   [[include:auth.md]]
  */
 export class AppConfig {
   /**
    * Blockstack apps are uniquely identified by their app domain.
    * @type {string}
    */
-  appDomain: string
+  public appDomain: string
```

Comment:
I originally suggested we use these `public`/`private` modifiers for a couple reasons:
1) They are a recommended eslint/tslint rule (currently not enabled), which we may want in the future.
2) I originally thought it was one of the only ways to control some parts of the docgen output (before finding out about the `@ignore` feature).

But now, I think we should hold off on introducing them for a couple reasons. Typescript enforces `private` so it controls how other typescript projects can use this lib, and `public` gives devs strong indication of intended usage/purpose. 

I think we'll be in a better position to do something like enable the eslint flag and go through the codebase to make these decisions once we have some of our other typescript repos (and 3rd parties) using this lib.

Reply:
OK. I'll make those changes.